docs/application/dotnet/guides/nui/widget.md

```diff
@@ -0,0 +1,254 @@
+# NUI Widget Application
```

Comment:
Excuse me, can you tell me what means "toc file"?

Reply:
@tscholb, since it is a new file, we require to update the table of contents (toc) file.

https://github.com/Samsung/tizen-docs/blob/d2a99fcbf74263f2f056000359a03db1f9413dcb/docs/application/toc_all.md

Hope this helps.

Reply:
Thank you for your kind explanation.
I will update it  around next week.